app/mailers/passenger_mailer.rb

```diff
@@ -3,4 +3,14 @@ def notify_archived(passenger)
     @passenger = passenger
     mail to: @passenger.email, subject: 'Special Transit Account Archived'
   end
+
+  def send_brochure(passenger)
+    @passenger = passenger
+    mail to: @passenger.email, subject: 'Your UMass Transit Account'
+  end
+
+  def notify_pending(passenger)
+    @passenger = passenger
+    mail to: @passenger.email, subject: 'UMass Transit Account Created'
```

Comment:
I just noticed the subject line--while the spec trans people may work out of the same building as Umass Transit, the department is actually "Special Transportation".
So the subject line should read "Special Transportation Account Created"